src/App.tsx

```diff
@@ -1,26 +1,72 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { setTodos } from './features/todos';
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const dispatch = useAppDispatch();
+  const todos = useAppSelector(state => state.todos);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const { status, query } = useAppSelector(state => state.filterTodos);
+
+  const [isLoadingTodos, setIsLoadingTodos] = useState<boolean>(false);
+
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  useEffect(() => {
+    const fetchTodos = async () => {
+      try {
+        setIsLoadingTodos(true);
+        const fetchedTodos: Todo[] = await getTodos();
+
+        dispatch(setTodos(fetchedTodos));
```

Comment:
better use async actions for fetching date
you can read more here https://redux-toolkit.js.org/api/createAsyncThunk